public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -65,6 +65,9 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
       pluginVersion: libPanelModel.pluginVersion,
       displayMode: libPanelModel.transparent ? 'transparent' : undefined,
       description: libPanelModel.description,
+      // To be replaced with it's own option persisted option instead derived
+      hoverHeader: !libPanelModel.title && !libPanelModel.timeFrom && !libPanelModel.timeShift,
```

Comment:
the panel title can be different from lib panel name, is this the dashboard panel title or lib panel name?

Reply:
this is dashboard panel title
<img width="1202" alt="Screenshot 2024-05-09 at 17 56 10" src="https://github.com/grafana/grafana/assets/37613182/af952989-3850-459f-8a9e-e29cd428b983">

Reply:
@Sergej-Vlasov ok great